tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I think you need to change it here to es6 to be able to use async/await and decorators but I might be wrong on that